docs/new-rule.md

```diff
@@ -15,10 +15,7 @@ Use the [`astexplorer` site](https://astexplorer.net) with the `espree` parser a
 
 - Go to the `test` directory and duplicate the `no-hex-escape.js` file and rename it to the name of your rule. Then write some tests before starting to implement the rule.
 - Go to the `rules` directory and duplicate the `no-hex-escape.js` file and rename it to the name of your rule. Then start implementing the new rule logic.
-- Don't forget to add `meta.type` to rule(especially if it's fixable). Possible types are:
-  - `problem` - the rule flags a potential problem with your code.
-  - `suggestion` - the rule flags something that could be done in a better or preferred way.
-  - `layout` - the rule flags changes limited to whitespace, semicolons, commas, or parentheses.
+- Add `meta.type` to rule.
```

Comment:
As commented in https://github.com/sindresorhus/eslint-plugin-unicorn/pull/241#discussion_r254770269, this should link to the ESLint docs.

Reply:
Ok, added.
Though at this point it is getting a bit stupid, as this link is the first line of the whole document. I intended this to be a quick helper to returning contributors who didn't know about the new meta.type property so they can fix code without going anywhere, and now it is just a duplicate of part of docs with a duplicate of link to said docs.